wled00/wled.cpp

```diff
@@ -1010,8 +1010,17 @@ void WLED::handleConnection()
     }
   } else if (!interfacesInited) { //newly connected
     DEBUG_PRINTLN();
-    DEBUG_PRINT(F("Connected! IP address: "));
-    DEBUG_PRINTLN(Network.localIP());
+    DEBUG_PRINTLN("");
+    DEBUG_PRINT(F("Connected! IP address: http://"));
+    DEBUG_PRINT(Network.localIP());
+    if (Network.isEthernet()) {
```

Comment:
Unless network.cpp is updated this will return true if static IP is set.
I will disconnect WiFi in such case even if link is not up.